api/v1alpha1/spacerequest_types.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SpaceRequestSpec defines the desired state of Space
+// +k8s:openapi-gen=true
+type SpaceRequestSpec struct {
+	// TierName is a required property introduced to retain the name of the tier
+	// for which this Space is provisioned
+	// If not set then the tier name will be set automatically
+	// +optional
+	TierName string `json:"tierName,omitempty"`
```

Comment:
in the comment, you mentioned that the property is required, but you added the `// +optional` and `omitempty` markers, so it's not consistent. 
let's make it required for now until we have a nice way of handling default tiers for sub-spaces.
```suggestion
	// TierName is a required property introduced to retain the name of the tier
	// for which this Space is provisioned
	// If not set then the tier name will be set automatically
	TierName string `json:"tierName"`
```

Reply:
fixed in: [04797c2d7449e7](https://github.com/codeready-toolchain/api/pull/333/commits/04797c2d7449e763fe1aa2f83a3058b5951df2c8)

Now that I'm checking we have other CRDs with this same small issue. thx!